src/runTests.js

```diff
@@ -5,8 +5,16 @@ const server = require('../server');
 const { existsSync } = require('fs');
 const { spawn, execFileSync, execSync } = require('child_process');
 
+// This is the default value for boot timeout
+const TWO_MINUTES = 2;
```

Comment:
but if we change the integer, we have to change the variable name too! in two places! 

```
// Default boot timeout in minutes
const BOOT_TIMEOUT = 2
```